src/Middleware/RequestThrottling/test/MiddlewareTests.cs

```diff
@@ -72,5 +72,14 @@ public async Task QueuedRequestsContinueWhenSpaceBecomesAvailible()
             await t1.OrTimeout();
             await t2.OrTimeout();
         }
+
+        [Fact]
+        public void InvalidArgumentIfMaxConcurrentRequestsIsNull()
+        {
+            Assert.Throws<ArgumentException>(() =>
```

Comment:
Once you update the parameters passed into the exception (see earlier comment), I believe you can add a check to this assertion that the right parameter name was passed in (`options`).

Reply:
Yes, Thows returns the exception object and you can inspect its properties.